go.mod

```diff
@@ -12,6 +12,6 @@ require (
 	github.com/mitchellh/hashstructure v1.1.0
 	github.com/onsi/gomega v1.18.1
 	github.com/xanzy/go-gitlab v0.55.1
-	golang.org/x/crypto v0.0.0-20210616213533-5ff15b29337e
+	golang.org/x/crypto v0.0.0-20220214200702-86341886e292 // indirect
```

Comment:
Looks like this is not needed anymore

Reply:
This crypto dependency was always there; it was just hidden before. Starting in Go 1.17 you are required to include all indirect dependencies in the go.mod anyway. This go.mod is still using 1.16 as the min Go version.

Reply:
Exactly, you'll see that "change" in `go.sum`. I'll leave it as is, since it's marked as `// indirect` anyways.